src/objects.rs

```diff
@@ -1967,6 +1968,7 @@ pub struct Occupancy {
         serialize_with = "ser_from_opt_bool"
     )]
     pub sunday: Option<bool>,
+    #[derivative(PartialEq = "ignore")]
```

Comment:
This is weird. Ignoring this field for equality, is incorrect to me. I guess you're just looking at the problem from one particular angle where it does make sense. If you need a specific need for comparing occupancies, maybe you need a specific function, instead of changing the general `PartialEq` implementation.